docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -167,8 +167,11 @@ The annotation `nginx.ingress.kubernetes.io/affinity-mode` defines the stickynes
 
 If you use the ``cookie`` affinity type you can also specify the name of the cookie that will be used to route the requests with the annotation `nginx.ingress.kubernetes.io/session-cookie-name`. The default is to create a cookie named 'INGRESSCOOKIE'.
 
-The NGINX annotation `nginx.ingress.kubernetes.io/session-cookie-path` defines the path that will be set on the cookie. This is optional unless the annotation `nginx.ingress.kubernetes.io/use-regex` is set to true; Session cookie paths do not support regex.
+The NGINX annotation `nginx.ingress.kubernetes.io/session-cookie-path` defines the path that will be set on the cookie. This is optional unless the annotation `nginx.ingress.kubernetes.io/use-regex` is set to true; Session cookie paths do not support regex. Use `session-cookie-path` to configure additional cookie attributes, e.g.
 
+```yaml
+nginx.ingress.kubernetes.io/session-cookie-path: "/; SameSite=Lax"
```

Comment:
I like don't like this approach. Please check the code that interacts with the annotation https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/lua/balancer/sticky.lua#L41

Reply:
Can you be more specific? I have read the code, and this is currently the only way to achieve samesite attribute on the cookie. Or have I overlooked something entirely?

Reply:
To clarify, I know this is not the intended use of the path variable. This PR is merely to document how users can currently prepare for Chrome 81, while a less hacky solution is developed.

Reply:
To clarify, I know this is not the intended use of the path variable. This PR is merely to document how users can currently prepare for Chrome 81, while a less hacky solution is developed.